Makefile.release

```diff
@@ -51,7 +51,8 @@ GITHUB:=coredns
 DOCKER:=coredns
 DOCKER_IMAGE_NAME:=$(DOCKER)/$(NAME)
 GITCOMMIT:=$(shell git describe --dirty --always)
-LINUX_ARCH=amd64 arm arm64 ppc64le s390x
+PLATFORMS:=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
+LINUX_ARCH:=$(shell echo $(PLATFORMS) | sed -e "s/linux\///g" | tr "," " ")
```

Comment:
isn't there a gnu make builtin for this? Either to create the PLATFORMS list or vice versa?